### ISSUE TITLE
jbfavre/vertica:latest doesn't work

### ISSUE THREAD

**Issue by b13f** [jbfavre/docker-vertica#3]
\# docker run -p 5433:5433 --name vertica jbfavre/vertica:latest
Starting up
Fixing filesystem permissions
Creating database
Info: no password specified, using none
Database with 1 or 2 nodes cannot be k-safe and it may lose data if it crashes
Distributing changes to cluster.
Error: Not all nodes could be reached during database creation
Could not reach node 127.0.0.1
Shutting Down
Vertica: Closing active sessions
vsql: could not connect to server: Connection refused
	Is the server running on host "???" and accepting
	TCP/IP connections on port 5433?

jbfavre/vertica:8.1 works fine

**Comment by jbfavre** [comment]
Thanks for the report, I will have a look at it ASAP

**Comment by jbfavre** [comment]
Fixed in e57d65f

Thanks for the report !

**Comment by anonymouskitler** [comment]
Having a similar issue @jbfavre.
```
Starting up
Fixing filesystem permissions
Starting Database
Info: no password specified, using none
Database docker is not defined. Defined databases []
```

**Comment by jbfavre** [comment]
@chandureddivari please consider opening a distinct issue and provide as much details as you reasonably can (image version, start command, arguments, etc…)

**Comment by rimoore** [comment]
Same problem myself. I can only get the 9,0 tag running.

**Comment by jbfavre** [comment]
@chandureddivari I've fixed some issues in 9.2.0-7 as of yesterday
Please try this version.

If it still doesn't work, please open a new issue (this one has been closed long ago) and provide as many details as you can like logs, docker version, image version, etc…

**Comment by anonymouskitler** [comment]
@jbfavre Thanks for the fix. It's working now. 🙏
